website/docs/r/private_endpoint.html.markdown

```diff
@@ -226,6 +234,8 @@ A `private_service_connection` block exports:
 
 * `private_ip_address` - (Computed) The private IP address associated with the private endpoint, note that you will have a private IP address assigned to the private endpoint even if the connection request was `Rejected`.
 
+* `network_interface_id` - (Computed) The private IP address associated with the private endpoint, note that you will have a private IP address assigned to the private endpoint even if the connection request was `Rejected`.
```

Comment:
I can't find this attribute listed in the `private_service_connection` block. Did we want to get that added or should we remove this line from the docs?

Reply:
@mbfrahry - Good call, that was some leftover copy pasta as I was trying to figure out the "right" way/place to export these attributes. I've amended the last commit to remove that from the docs and keep everything clean(er).